docs/docs/concepts/embedding_models.mdx

```diff
@@ -6,11 +6,11 @@
 * [Documents](/docs/concepts/retrievers/#interface)
 
 :::
+
 :::info[Note]
+This conceptual overview focuses on text-based embedding models.
 
-This conceptual overview focuses on text-based embedding models. 
-Embedding models can also be [multi-modal](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-multimodal-embeddings), allowing for the embedding of a wide variety of data types.
- 
+Embedding models can also be [multimodal](/docs/concepts/multimodality) though such models are not currently supported by LangChain.
```

Comment:
we do have Open CLIP in experimental, but still probably fine to say they are not supported in LangChain:
https://github.com/langchain-ai/langchain-experimental/tree/main/libs/experimental/langchain_experimental/open_clip